Midterm/tip calculate/tip.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<! add information about html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Tip Calculator</title>
+    <link rel="stylesheet" type="text/css" href="tip.css">
+    <script type="text/javascript" src="tip.js"></script>
+</head>
+
+<body>
+    <! body include header, main, footer>
+
+    <header>
+        <h1>Tip Calculator</h1>
+    </header>
+
+    <! add main contents in main>
+    <main>
+        <div id="wrapper">
+            <form action="javascript:void(calc())">
```

Comment:
Using action is fine for the case like this, it's generally more recommended to use click event listener.